llvm/test/lab3/kulikov_artem/vfmadd.mir

```diff
@@ -0,0 +1,497 @@
+# RUN: llc -mtriple x86_64-unknown-linux-gnu -run-pass=x86-kulikov-fma  %s -o - | FileCheck %s
+
+# src.c
```

Comment:
What output will be for the following example?
```
# __m128d foo(__m128d a, __m128d b, __m128d c, __m128d d) { 
 #     __m128d tmp = a * b; 
 #     c = tmp + b;
 #     d = tmp + c; 
 #     return d;
```

Reply:
%1:vr128 = COPY $xmm1
%0:vr128 = COPY $xmm0
%4:vr128 = nofpexcept MULPDrr %0, %1, implicit $mxcsr
%5:vr128 = nofpexcept ADDPDrr %4, %1, implicit $mxcsr
%6:vr128 = nofpexcept ADDPDrr %4, killed %5, implicit $mxcsr
$xmm0 = COPY %6
RET 0, $xmm0